get_lunch.sh

```diff
@@ -13,7 +13,7 @@ service="image"
 id="lunch"
 destination="~/lunch/tu-$id-$date.jpg"
 # https://www.tu.no/?module=TekComics&service=image&id=lunch&key=2020-05-28
-curl --get --url "$url" --data-urlencode "module=$module"  --data-urlencode "service=$service" --data-urlencode "id=$id" --data-urlencode "key=$date" --output "$destination"
+curl --get --url "$url$date" --data-urlencode "module=$module"  --data-urlencode "service=$service" --data-urlencode "id=$id" --data-urlencode "key=$date" --output "$destination"
```

Comment:
Is it not better to use the `--data-urlencode` directives to ensure that url queries are handled properly? 🤔